src/server/main.py

```diff
@@ -30,13 +31,14 @@ def _get_taxon(params, headers):
         'type': 'object',
         'required': ['id'],
         'properties': {
-            'id': {'type': 'string', 'pattern': id_pattern}
+            'id': {'type': 'string', 'pattern': id_pattern},
+            'ts': {'type': 'integer', 'minimum': 0},
         }
     }
     jsonschema.validate(instance=params, schema=schema)
-    (coll, key) = params['id'].split('/')
-    params['key'] = key
-    del params['id']
+    (coll, _id) = params['id'].split('/')
```

Comment:
I'm really not a fan of exposing the collection names outside the RE API. That means the collections can never be renamed, which means things like red/green updates are impossible.

Reply:
The collections don't actually get used in the query. We could set up aliases soon. But for now, I'll leave things as-is so it can get functional.